src/game/GameManager.h

```diff
@@ -98,6 +98,7 @@ class GameManager {
 
     int32_t addObject(const Object&);
     void deleteObject(const int32_t id);
+    Object& getBase();
```

Comment:
Trista has a PR up that I think handles this
I'd suggest we get that merged in first, so that you're not redoing her work